tests/Domain/Integrations/Repositories/EloquentUdbOrganizerRepositoryTest.php

```diff
@@ -79,4 +79,11 @@ public function testDelete(): void
             'organizer_id' => $this->organizer1->organizerId,
         ]);
     }
+
+    public function test_it_can_get_an_udb_organizer_by_id(): void
```

Comment:
Not related to this pull request, but I think the `testDelete` method doesn't create the `$organizer1` inside the database.

Reply:
haha, good catch, will fix it.